src/schema.ts

```diff
@@ -273,44 +277,17 @@ export interface FlywheelConnectedEvent {
 }
 
 /**
- * Represents every channel that can be played on the vibraphone.
- *
- * We decided to start at one since the [[BassString]] type also
- * starts at one. We also decided to only represent playable
- * channels and not actual channels (two per note) because that is
- * an implementation detail of the physical MMX.
```

Comment:
I'm not sure why this is mentioned since most of the MMX details, from the inclusion of instruments to the size of the programming wheel, effect this schema. Why is the bass limited to 4 strings? Because that is an implementation detail of the MMX.

Reply:
The model is meant to match the actual mmx configuration, the number of bass strings wrong change so why support making that configurable when locking it down can better catch user errors?

Reply:
I agree. I was trying to point out that it was unnecessary in the description.